wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Insights > Most Popular Day and Hour

Fixes #10515 

To test:
- On a site with most popular views, go to Stats > Insights.
- Verify there is a `MOST POPULAR DAY AND HOUR` card, like so:

<img width="400" alt="most_popular" src="https://user-images.githubusercontent.com/1816888/48873863-7e5dc600-edad-11e8-9c76-f3f111534df9.png">

- On a site with _no_ most popular views, go to Stats > Insights.
- Verify there is an empty `MOST POPULAR DAY AND HOUR` card, like so:

<img width="400" alt="no_data" src="https://user-images.githubusercontent.com/1816888/48874613-66884100-edb1-11e8-9994-0ca51ea0f3f8.png">

NOTE: You may notice this card has no bottom margin. More specifically, the last card, whatever it is, will have no bottom margin. That's not new, it's just more obvious as more cards are added. It shall be address later!
